### PR TITLE
Add a bit more info to the postinstall message

### PR DESCRIPTION
**Pull request by zero-24**
this add a bit more info to the postinstall message for the stats plugin.

Text is from here: https://developer.joomla.org/about/stats.html

Now it looks like: 
a_bit_more_info

### CODE REVIEW

**Comment by zero-24** [comment]
Thanks @wilsonge

**Comment by wilsonge** [comment]
Thankyou!

File: administrator/language/en-GB/en-GB.com_cpanel.ini [hunk 1]
```diff
@@ -27,7 +27,7 @@ COM_CPANEL_MSG_PHPVERSION_BODY="Beginning with Joomla! 3.3, the version of PHP t
...
+COM_CPANEL_MSG_STATS_COLLECTION_BODY="Since Joomla 3.5 Joomla contains an anonymous statistics tracking plugin that provides your Joomla version, PHP version, database engine and version, and server operating system. This data is collected in order to ensure that as we develop new versions of Joomla we provide the most optimal software taking advantage of the latest database and PHP features but without loosing significant numbers of users. It became especially clear this was required after the requirement of PHP 5.3.10 was implemented after the release of Joomla 3.3 for implementation of BCrypt passwords.<br><br><a href="_QQ_"https://developer.joomla.org/about/stats.html"_QQ_">We are making this data publicly available</a> through our API and we will be providing graphs representing the Joomla version, PHP versions and database engines of sites reporting their data in the interest of transparency to the community and to help 3rd party developers.<br><br>If you do not wish to provide the Joomla! project with this information you can simply disable the plugin called System - Joomla! Statistics"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_cpanel.ini]: This string fails pretty badly for readability and in parts breaks the style guides.

Comment (wilsonge) [inline administrator/language/en-GB/en-GB.com_cpanel.ini]: PR it and I'll merge it for beta 2. I know it isn't the best which is why I initially was only linking to the new stats page.

**Comment by brianteeman** [comment]
This is very complex english and hard to both read and understand and in
parts breaks our own style guide on using the ! with joomla

On 6 November 2015 at 23:53, George Wilson <notifications@github.com> wrote:

> Thankyou!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/8302#issuecomment-154580200>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by brianteeman** [comment]
There are also some obvious spelling and grammatical errors

On 6 November 2015 at 23:56, Brian Teeman <brian@teeman.net> wrote:

> This is very complex english and hard to both read and understand and in
> parts breaks our own style guide on using the ! with joomla
>
> On 6 November 2015 at 23:53, George Wilson <notifications@github.com>
> wrote:
>
>> Thankyou!
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/joomla/joomla-cms/pull/8302#issuecomment-154580200>.
>>
>
>
>
> --
> Brian Teeman
> Co-founder Joomla! and OpenSourceMatters Inc.
> http://brian.teeman.net/
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by zero-24** [comment]
It is just copied from https://developer.joomla.org/about/stats.html Let me know how it can be improved (you know i'm not a native en-GB speaker) and i can PR it if you don't want do it your self ;)

**Comment by brianteeman** [comment]
Try this. 


Since Joomla 3.5 a statistics plugin will submit anonymous data to the Joomla Project. This will only submit the Joomla version, PHP version, database engine and version, and server operating system. 

This data is collected to ensure that future versions of Joomla can take advantage of the latest database and PHP features without effecting significant numbers of  users. The need for this became clear when a minimum of PHP 5.3.10 was required when Joomla 3.3 implemented the more secure BCrypt passwords.

In the interest of full transparency and to help developers this data is publicly available. An API and graphs will show the Joomla version, PHP versions and database engines in use. 

If you do not wish to provide the Joomla Project with this information you can disable the plugin called System - Joomla Statistics.

**Comment by brianteeman** [comment]
updated - missed the fact the styleguide says to use the! when referring to specific versions and I added one too many uppercase characters in Bcrypt.

Since Joomla! 3.5 a statistics plugin will submit anonymous data to the Joomla Project. This will only submit the Joomla version, PHP version, database engine and version, and server operating system.

This data is collected to ensure that future versions of Joomla can take advantage of the latest database and PHP features without effecting significant numbers of users. The need for this became clear when a minimum of PHP 5.3.10 was required when Joomla! 3.3 implemented the more secure Bcrypt passwords.

In the interest of full transparency and to help developers this data is publicly available. An API and graphs will show the Joomla version, PHP versions and database engines in use.

If you do not wish to provide the Joomla Project with this information you can disable the plugin called System - Joomla Statistics.

**Comment by zero-24** [comment]
Thanks Brian see: https://github.com/joomla/joomla-cms/pull/8303

**Pull request merged**